src/wazuh_db/schema_vuln_detector.sql

```diff
@@ -129,6 +131,34 @@ CREATE INDEX IF NOT EXISTS IN_VAR_ID ON VARIABLES (VID);
 CREATE INDEX IF NOT EXISTS IN_VAR_VALUE ON VARIABLES (VALUE);
 CREATE INDEX IF NOT EXISTS IN_VAR_TARGET ON VARIABLES (TARGET);
 
+CREATE TABLE IF NOT EXISTS DEPENDENCIES (
+    ID TEXT NOT NULL,
+    NAME TEXT NOT NULL,
+    OPERATION TEXT NOT NULL,
+    OPERATION_VALUE TEXT NOT NULL,
+    TARGET TEXT NOT NULL REFERENCES VULNERABILITIES_INFO (V_OS),
+    INSTALLED INTEGER DEFAULT 0,
+    PRIMARY KEY(ID, NAME, OPERATION, OPERATION_VALUE, TARGET)
+);
+CREATE INDEX IF NOT EXISTS IN_VAR_ID ON DEPENDENCIES (ID);
```

Comment:
It is needed to create indexes for all the fields? I think it would be better to include them only for fields that will be iterated in searches such as the name and target.